docs/source/guide/prompts_keys.md

```diff
@@ -0,0 +1,106 @@
+
+* In one scenario, you get one provider connection per organization, and this provides access to a set of whitelisted models. Examples include:
+
+    * OpenAI
+    * Vertex AI
+    * Gemini
+
+* In the second scenario, you add a separate API key per model. Examples include:
+
+    * Azure OpenAI
+    * Custom
+
+Once a model is added via the API key, anyone in the organization who has access to the Prompts feature can select the associated models when executing their prompt. 
+
+You can see what API keys you have and add new ones by clicking **API Keys** in the top right of the Prompts page to open the **Model Provider API Keys** window:
+
+![Screenshot of the API keys button](/images/prompts/model_keys.png)
+
+## OpenAI API key
+
+You can only have one OpenAI key per organization. This grants you access to set of whitelisted models. For a list of these models, see [Supported base models](prompts_overview#Supported-base-models). 
+
+If you don't already have one, you can [create an OpenAI account here](https://platform.openai.com/signup).
+
+You can find your OpenAI API key on the [API key page](https://platform.openai.com/api-keys). 
+
+Once added, all supported models will appear in the base model drop-down when you [draft your prompt](prompts_draft).
+
+## Gemini API key
+
+You can only have one Gemini key per organization. This grants you access to set of whitelisted models. For a list of these models, see [Supported base models](prompts_overview#Supported-base-models). 
+
+For information on getting a Gemini API key, see [Get a Gemini API key](https://ai.google.dev/gemini-api/docs/api-key).
+
+Once added, all supported models will appear in the base model drop-down when you [draft your prompt](prompts_draft).
+
+## Vertex AI JSON credentials
+
+You can only have one Vertex AI key per organization. This grants you access to set of whitelisted models. For a list of these models, see [Supported base models](prompts_overview#Supported-base-models). 
+
+Follow the instructions here to generate a credentials file in JSON format: [Authenticate to Vertex AI Agent Builder - Client libraries or third-party tools](https://cloud.google.com/generative-ai-app-builder/docs/authentication#client-libs)
+
+The JSON credentials are required. You can also optionally provide the project ID and location associated with your Google Cloud Platform environment. 
+
+Once added, all supported models will appear in the base model drop-down when you [draft your prompt](prompts_draft).
+
+## Azure OpenAI key
+
+Each Azure OpenAI key is tied to a specific deployment, and each deployment comprises a single OpenAI model. So if you want to use multiple models through Azure, you will need to create a deployment for each model and then add each key to Label Studio. 
+
+For a list of the Azure OpenAI models we support, see [Features, requirements, and constraints](prompts_overview#Features-requirements-and-constraints). 
```

Comment:
should this point to the Supported base model section like the other providers do?